packages/forms/src/UIForm/Message/generateId.js

```diff
@@ -0,0 +1,16 @@
+import uuid from 'uuid';
```

Comment:
Since this is used by other form elements than Message, should't it be somewhere else ?